cfme/tests/configure/test_visual_cloud.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+
+
+import pytest
+import re
+from cfme import login
+from cfme.configure.settings import visual
+from cfme.fixtures import pytest_selenium as sel
+from cfme.web_ui import paginator, toolbar as tb, menu
+from utils.conf import cfme_data
+from utils.providers import setup_a_provider as _setup_a_provider
+
+
+@pytest.fixture(scope="module")
+def setup_a_provider():
+    _setup_a_provider(prov_class="cloud", validate=True, check_existing=True)
+
+
+@pytest.yield_fixture(scope="module")
+def set_grid():
+    visual.grid_view_limit = 5
+    yield
+    visual.grid_view_limit = 20
```

Comment:
I think those attributes are doing the getter/setter thing, so it could be as easy as this:
```python
grid_view_limit = visual.grid_view_limit
visual.grid_view_limit = 5
yield
visual.grid_view_limit = grid_view_limit